From 046fcc221cbe4897a3c033009ce9cb3a8a668d02 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hmehrtens@maxlinear.com>
Date: Thu, 10 Dec 2020 10:36:49 +0100
Subject: [PATCH] PONRTSYS-8285: Fix compile warning in arp_ndp_reinsert.ko

This fixes a compile warning introduced when this kernel module was
added.

Fixes: 429bb65bcdd4 ("PONRTSYS-8285: Add arp_ndp_reinsert.ko")
---
 drivers/net/datapath/bridge_filter/arp_ndp_reinsert.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/datapath/bridge_filter/arp_ndp_reinsert.c b/drivers/net/datapath/bridge_filter/arp_ndp_reinsert.c
index 4961b39748a7..1712b6fc81a3 100644
--- a/drivers/net/datapath/bridge_filter/arp_ndp_reinsert.c
+++ b/drivers/net/datapath/bridge_filter/arp_ndp_reinsert.c
@@ -102,8 +102,8 @@ static unsigned int arp_ndp_reinsert_hook_arp(struct sk_buff *skb, int nhoff)
 
 	ret = arp_ndp_reinsert_packet(skb);
 	if (ret) {
-		netdev_err(skb->dev, "%s: reinsert fail %i, accepted\n", ret,
-			   __func__);
+		netdev_err(skb->dev, "%s: reinsert fail %i, accepted\n",
+			   __func__, ret);
 		return NF_ACCEPT;
 	}
 	return NF_DROP;
@@ -161,7 +161,7 @@ static unsigned int arp_ndp_reinsert_hook_ipv6(struct sk_buff *skb, int nhoff)
 		ret = arp_ndp_reinsert_packet(skb);
 		if (ret) {
 			netdev_err(skb->dev, "%s: reinsert fail %i, accepted\n",
-				   ret, __func__);
+				   __func__, ret);
 			return NF_ACCEPT;
 		}
 		return NF_DROP;
